src/components/admin/user-profile-card/UserProfileCard.scss

```
@@ -42,85 +42,32 @@
 		&:hover {
 			background-color: var(--s-color-fill-default-hover);
 		}
+	}
 
-		&-wrapper {
-			display: flex;
-			gap: var(--s-spacing-nano);
-			align-items: center;
-			position: relative;
-			cursor: pointer;
-			padding: var(--s-spacing-nano) var(--s-spacing-x-small);
-			width: 100%;
-			border-radius: var(--s-border-radius-small);
-		}
-
-		&-avatar {
-			width: var(--input-height);
-			height: var(--input-height);
-			align-items: center;
-			justify-content: center;
-			border-radius: 100%;
-			overflow: hidden;
-			border: solid 2px var(--s-color-border-highlight);
-			display: inline-flex;
-			font-weight: 600;
-			text-align: center;
-			color: var(--s-color-content-light);
-			text-transform: uppercase;
-			position: relative;
-
-			&::after {
-				content: '';
-				border: solid 3px #fff;
-				position: absolute;
-				top: 0;
-				right: 0;
-				bottom: 0;
-				left: 0;
-				z-index: var(--s-index-max);
-				border-radius: 100%;
+	@include darkmode {
+		.dropdown-item {
+			&-caption {
+				color: var(--s-color-content-dark);
 			}
+			.ui-dropdown-item {
+				color: var(--s-color-content-dark);
 
-			img {
-				max-width: 100%;
-				padding: 2px;
+				&:hover {
+					background-color: var(--s-color-fill-dark-hover);
+					color: var(--s-color-content-dark);
+				}
 			}
 		}
-
-		&-text {
-			display: flex;
-			flex: 1;
-			align-items: center;
-			justify-content: space-between;
-			line-height: 1.2;
-			text-align: left;
-
-			.name {
-				display: block;
-				font-weight: bold;
+		.user-card {
+			&-btn {
+				&:hover,
+				&:active {
+					background: var(--dark-100);
```

essa variável é antiga, podemos substituir por uma nova, acredito que essa possa ser a ideal:
```suggestion
					background: var(--s-color-fill-default-hover);
```